back-end/src/sciler/communication/communication_test.go

```diff
@@ -4,62 +4,11 @@ import (
 	"errors"
 	"fmt"
 	"github.com/eclipse/paho.mqtt.golang"
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 	"testing"
 	"time"
 )
 
-func TestNewCommunicator(t *testing.T) {
```

Comment:
why is this not tested anymore?

Reply:
It is, through other tests.
The only parts that are now covered are harder to test, aka the callbacks for connect/reconnect/disconnect. If you know a good way of testing those, let me know :)